app/assets/javascripts/Components/markus_file_manager.jsx

```diff
@@ -38,6 +38,11 @@ class RawFileManager extends RawFileBrowser {
     }
   };
 
+  upload_or_submit_file = () => {
```

Comment:
This function name is a bit misleading, it sounds like this is the action that causes a file to be uploaded.

Add the suffix `_label` so that it's clear what's being returned.